.travis.yml

```diff
@@ -22,6 +24,19 @@ script:
    - pushd samples/compile-timeDI/ && sbt ++$TRAVIS_SCALA_VERSION test && popd
    - pushd samples/runtimeDI/ && sbt ++$TRAVIS_SCALA_VERSION test && popd
 
+jobs:
+  include:
+    - stage: deploy
+      name: "TEMP Publish artifacts to Bintray"
+      script: sbt +mimaReportBinaryIssues
+      scala: 2.13.1
+      env: TRAVIS_JDK=11
+
+stages:
+  - name: test
+  - name: publish
+#    if: ((branch = master AND type = push) OR (tag IS present)) AND NOT fork
```

Comment:
why comment this out?

should it be called 'publish' in the 'jobs' list also?

Reply:
Just testing... great that you saw the name mismatch.